math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -1143,6 +1143,23 @@ RVec<T> Take(const RVec<T> &v, const int n)
    return r;
 }
 
+/// Return a copy of the container without the elements at the specified indices.
+template <typename T>
+RVec<T> Drop(const RVec<T> &v, RVec<typename RVec<T>::size_type> idxs)
+{
+   RVec<T> r;
+   r.reserve(v.size() - idxs.size());
+   std::sort(idxs.begin(), idxs.end());
+   auto discardIt = idxs.begin();
+   for (auto i = 0u; i < v.size(); ++i) {
+      if (i == *discardIt)
+         ++discardIt;
```

Comment:
I think it would be good to have a `std::unique` after the `std::sort` (and `r.reserve(v.size() - idxs.size());` only after that) to protect against duplicate indices. Right now, I think the loop would get stuck

Reply:
ah, of course thanks :)